utils/utils.go

```diff
@@ -93,6 +93,14 @@ func SetEnvVariables(variables map[string]string) error {
 	return nil
 }
 
+// GetEnvOrElse get the value from the OS environment or use the else value if variable is not present.
+func GetEnvOrElse(name, orElse string) string {
+	if value, ok := os.LookupEnv(name); ok {
+		return value
+	}
+	return orElse
+}
+
```

Comment:
Not sure if this is required but... shouldn't we check whether the envVar value is also is non-empty and/or non-blank?